presto-benchto-benchmarks/src/main/resources/sql/presto/tpcds/q02.sql

```diff
@@ -21,13 +21,13 @@ UNION ALL (
 , wswscs AS (
```

Comment:
Can you describe in the commit message why was that and why now this modification can be removed?